tests/expected_runtime/k8s_tests_expected_runtime.csv

```diff
@@ -55,6 +55,7 @@ test,run_time(seconds)
 "k8s_cmdline_tests.yaml, query name: test25-expl",1.08
 "k8s_cmdline_tests.yaml, query name: test4-expl",1.08
 "k8s_cmdline_tests.yaml, query name: istio-ingress-expl",4.1
+"k8s_cmdline_tests.yaml, query name: bad-yamls",0.02
```

Comment:
please update from 0.02 to a reasonable time

Reply:
done